.gitignore

```diff
@@ -6,4 +6,5 @@ node_modules/
 public/
 .deploy*/
 .idea
-.dccache
\ No newline at end of file
+.dccache
+_multiconfig.yml
```

Comment:
Where is this file?

Reply:
In the root, during npm run server eg. Configs are concatenated by hexojs by default if you set multimple configs during the run